src/core/config/border_router.h

```diff
@@ -75,6 +75,16 @@
 #define OPENTHREAD_CONFIG_BORDER_ROUTING_VICARIOUS_RS_ENABLE 1
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
+ *
+ * Define to 1 to enable Border Routing NAT64 support.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
+#define OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE 0
```

Comment:
This is by default disabled here and I see we have `OT_BORDER_ROUTING_NAT64` cmake option but we don't seem to enabling using this option in any of build scripts.

Do we have any CI builds on OT github actions that enables and covers this feature? (e.g., making sure it builds successfully under all toolchains, etc)?

Maybe we can enable it some of the build scripts `script/check-scan-build`, `script/cmake-build`, `script/make-pretty`?

Reply:
Enabled it in `script/check-scan-build` and `script/make-pretty`. Will enable it in `script/cmake-build` when I update the BR tests with this feature. Thanks!